app/drivers/sensor/pim447_trackball/Kconfig

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2021 Cedric VINCENT
+# SPDX-License-Identifier: Apache-2.0
+
+config ZMK_PIM447_TRACKBALL
+	bool "ZMK PIM447 trackball"
+	depends on I2C
+	help
+	  Enable I2C-based driver for Pimorono PIM447 trackball.
```

Comment:
Shouldn't it be "Pimoroni"?

Reply:
Oh right, thanks for the review!  I'll fix that when I rebase this driver on top of the mouse emulation branch once it gets integrated officially.